rabbitmq/autophrase/notification.py

```diff
@@ -62,132 +87,145 @@ def notification(toaddr, case, filename, links, sessionURL):
                 fpath[2] = 'Python NetworkX'
             elif fpath[2] == 'classification':
                 fpath[2] = 'Text Classification'
+        else:
+            fpath = []
 
-        if case == 0 or case == 'comment-fail':
-            html = """
-            <html> 
-                <head></head>
-                <body style="font-size:15px;font-fiamily:'Times New Roman', Times, serif;">
-            <div>
-                        <p>Dear user (session ID: """ + fpath[0] + """),</p>
-                <p>Your Reddit Comment collection has been terminated.</p>
-                <p>We are using the <b>id</b> and <b>permalink</b> from your Reddit Submission dataset
-                to collect comments and replies. It is most likely you have provide an incomplete Reddit Submission dataset missing these two fields.</p>
-                <p>Please try to reproduce the Reddit Submission with <b>id</b> and <b>permalink</b>, or switch to another dataset.</p>
-                <a href=""" + sessionURL + """>Go to your session...</a>
-                        <br>
-                        <p>Best Regards,</p>
-                        <p>Social Media Macroscope - SMILE </p>
-            </div>
-                </body>
-            </html>
-            """
-            subject = 'Your Reddit Comment collection has failed...'
-        elif case == 1 or case == 'comment-terminate':
-            html = """<html> 
-                        <head></head>
-                        <body style="font-size:15px;font-fiamily:'Times New Roman', Times, serif;">
-                            <div>
-                                <p>Dear user (session ID: """ + fpath[0] + """),</p>
-                                <p>Your Reddit Comment collection is exceeding 400 Megabyte, and is terminated due to lack of disk space.</p>
-                                <ul>
-                                    <li>You have requested comments and replies for the Reddit Submission (Post):<b>""" + \
-                   fpath[
-                       3] + """</b>. The partial comments we manage to collect and save will be compressed for you in an .zip file named <a href='""" + links + """'>""" + \
-                   fpath[3] + """-comments.zip</a> (click)</li>    
-                                    <li>In order to download this file, you need to first locate the original submission in the <b>Past Results</b> page in SMILE.
-                                       <a href=""" + reformat_sessionURL(sessionURL) + """>Go to your session.</a> 
+        if len(fpath)==3:
```

Comment:
When I tested, the file name was like `ywkim@illinois.edu/NLP/autophrase/dde72c7c-0d2c-4634-af9d-d94fb485d4f9/
So if the code splits this string using the '/', the length of the array would be 4 becoming like  `['ywkim@illinois.edu', 'Nature Language Processing', 'autophrase', 'dde72c7c-0d2c-4634-af9d-d94fb485d4f9', '']`
So if we change this like like `if len(fpath)>=3', it will work

Reply:
Should be fixed now. Good catch!